EIPS/eip-4750.md

```diff
@@ -103,6 +99,8 @@ In addition to container format validation rules above, we extend code section v
 3. `RJUMP`, `RJUMPI` and `RJUMPV` immediate argument value (jump destination relative offset) validation:
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
+4. First code section is invalid if it contains a `RETF` instruction.
+5. No unreachable sections are allowed, i.e. every section is referenced by at least one non-recursive `CALLF`, and section 0 is implicitly reachable.
```

Comment:
To reflect new aspect that was discovered recently
```suggestion
5. No unreachable sections are allowed, i.e. every section can be reached from section 0 with a series of `CALLF`/`JUMPF` instructions.
```

Reply:
1. as above, I would leave JUMPF out. I also don't think it's ok to mention an entire new opcode not introduced here, without linking 6206. And the latter we don't want to do for obvious reasons
2. I think we should leave "section 0 being reachable" in, if we're pedantic.